File: arrays/js/seller.js
const sellers = [
    {
      "name": "Barry Allen",
      "amount": 18196.0
    },
    {
      "name": "Logan",
      "amount": 4255.0
    },
    {
      "name": "Maria",
      "amount": 10298.0
    },
    {
      "name": "Ana",
      "amount": 26485.0
    },
    {
      "name": "Aurora",
      "amount": 19982.0
    },
    {
      "name": "Noah",
      "amount": 15820.0
    },
    {
      "name": "Leo",
      "amount": 21412.0
    },
    {
      "name": "Alex",
      "amount": 12000.0
    }
];

const findMaxSellerAmount = (arr) => {
    let maxAmount = 0;
    let seller;

    for (const item of arr) {
        if (item.amount > maxAmount) {
            maxAmount = item.amount;
            seller = item;
        }
    }

    return seller;
};

// Exemplo de uso
const maxSeller = findMaxSellerAmount(sellers);
console.log(maxSeller);

File: arrays/python/seller/seller.py
import json

def find_max_seller_amount(arr):
    max_amount = 0
    seller = None

    for item in arr:
        if item["amount"] > max_amount:
            max_amount = item["amount"]
            seller = item

    return seller

# Exemplo de uso
with open('file.json', 'r') as file:
    sellers = json.load(file)

max_seller = find_max_seller_amount(sellers)
print(max_seller)